src/js/services/settings.js

```diff
@@ -16,7 +16,7 @@ export default App.extend({
     const workspaceSettings = currentWorkspace.get('settings');
     const workspaceSetting = get(workspaceSettings, settingName);
 
-    if (workspaceSetting) return workspaceSetting;
+    if (!isUndefined(workspaceSetting)) return workspaceSetting;
```

Comment:
I think this is a bug I found where a workspace setting is sometimes set but doesn't take precedent as it should.

So in this settings scenario:

1. `manual_patient_creation: false` setting on the workspace level.
2. `manual_patient_creation: true` setting on the org level.

The workspace setting should be returned (`false`). 

But instead this `getSetting()` returns the org setting value because it's checking if the workspace setting is truthy. I think it should check if it's `undefined` instead, checking if the workspace setting exists or not.